server/routerlicious/kubernetes/routerlicious/templates/fluid-configmap.yaml

```diff
@@ -22,6 +22,7 @@ data:
         "lumberjack": {
             "options": {
                 "enableGlobalTelemetryContext": {{ .Values.lumberjack.options.enableGlobalTelemetryContext }}
+                "enableSanitization": {{ .Values.lumberjack.options.enableSanitization }}
```

Comment:
Are we reading this config anywhere? You might need to update `configureLumberjackLogging()` under logger.ts to read the new `lumberjack.options.enableSanitization` setting.

Reply:
I think it does, it gets injected to `Lumberjack` through the `options` object `Lumberjack.setup(config.engineList, config.schemaValidator, config.options);`

Reply:
You are right! I missed that part.

Reply:
Good point! You are totally right. I had missed that 😄